docs/guidance/map/beneficial_ownership.md

```diff
@@ -0,0 +1,103 @@
+# Beneficial ownership information
+
+This worked example describes how to:
+*   publish separate contracting and beneficial ownership datasets
+*   publish beneficial ownership data in OCDS
+
+The Beneficial Ownership Data Standard (BODS) [defines](http://standard.openownership.org/en/0.2.0/primer/whatisbo.html) “beneficial ownership” as:
+
+> The right to some share of a legal entity’s income or assets (ownership) or the right to direct or influence the entity’s activities (control).
+
+Publishing the beneficial owners of tenderers and suppliers can help users to:
+
+*   Identify collusion or deception
+*   Detect undeclared conflicts of interest
+*   Identify companies with debarred or sanctioned owners
+*   Assess risk
+
+For more information on the uses of beneficial ownership data in the context of procurement, see [Connecting the dots between beneficial ownership and procurement data](https://medium.com/opendatacoop/connecting-the-dots-between-beneficial-ownership-and-procurement-data-improving-processes-and-2fd0df9d9653). 
```

Comment:
Isn't this document much more relevant? https://www.openownership.org/uploads/OO%20BO%20Data%20in%20Procurement.pdf

Reply:
I guess so, @duncandewhurst what do you think?

Reply:
Yes! I wasn't aware of it.